application/src/main/resources/thingsboard.yml

```
@@ -521,6 +521,48 @@ queue:
     batch.size: "${TB_KAFKA_BATCH_SIZE:16384}"
     linger.ms: "${TB_KAFKA_LINGER_MS:1}"
     buffer.memory: "${TB_BUFFER_MEMORY:33554432}"
+    replication_factor: "${TB_QUEUE_KAFKA_REPLICATION_FACTOR:1}"
+    topic-properties:
+      rule-engine:
```

This Definition appears to be "huge" waste of space in the config. Maybe something like this:

```suggestion
      rule-engine: "${TB_QUEUE_KAFKA_RE_TOPIC_PROPERTIES:retention.ms:604800000;segment.bytes:26214400;retention.bytes:1048576000}"
```